src/css/profile/mobile/common/popup.less

```diff
@@ -98,9 +98,22 @@
 			padding-bottom: 24 * @unit_base;
 		}
 		input[type=checkbox] {
-			margin-top: 26 * @unit_base;
-			margin-bottom: -12 * @unit_base;
-			margin-right: 22 * @unit_base;
+			margin-left: 0 * @px_base;
+			margin-top: 0 * @px_base;
+			margin-bottom: 0 * @px_base;
+			margin-right: 12 * @px_base;
+			vertical-align: -10 * @px_base;
+			width: 32 * @px_base;
+			height: 32 * @px_base;
+		}
+		input[type=radio] {
+			margin-left: 0 * @px_base;
```

Comment:
the same as above.

And general rule:
You don't need `@px_base` with 0 eg.
`0 * @px_base` just simple `0`

Reply:
ok. I change below it.
margin: 0 12 * @px_base 0 0;